src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -61,6 +62,7 @@ def email() {
                 "ts-mtrotator": "ttsai@lsst.org",
                 "ts-observatory-control": "tribeiro@lsst.org",
                 "ts-observatory-model": "tribeiro@lsst.org",
+                "ts-ofc": "tribeiro@lsst.org, ttsai@lsst.org",
```

Comment:
You need to remove the space between the comma otherwise it won't work.

Reply:
Syntax change is not necessary in this case.